hsinstall fails to build

```
Preprocessing executable 'hsinstall' for hsinstall-2.7..                                                                            │········································
Building executable 'hsinstall' for hsinstall-2.7..                                                                                 │········································
[7 of 9] Compiling HSInstall.DeploymentInfo                                                                                         │········································
                                                                                                                                    │········································
/var/stackage/work/unpack-dir/unpacked/hsinstall-2.7-c382ecbd6ca77b6571b97bb136dc7b05b235541d3f9951188573edc2e3d6c9a9/src/app/HSInst│········································
all/DeploymentInfo.hs:30:5: error:                                                                                                  │········································
    Module                                                                                                                          │········································
    ‘Distribution.PackageDescription.Parsec’                                                                                        │········································
    does not export                                                                                                                 │········································
    ‘readGenericPackageDescription’                                                                                                 │········································
   |                                                                                                                                │········································
30 |   ( readGenericPackageDescription )                                                                                            │········································
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -3554,7 +3554,7 @@ packages:
 
     "Dino Morelli <dino@ui3.info> @dino-":
         - epub-metadata
-        - hsinstall < 0 # 2.7 https://github.com/commercialhaskell/stackage/issues/6763
+        - hsinstall
         - tce-conf
 
     "Jonathan Fischoff <jonathangfischoff@gmail.com> @jfischoff":

